LOCALIZATION.md

```diff
@@ -3,25 +3,51 @@
 Now that the first version of the cloud native glossary is live,
 we'd love to start localizing it into different languages. 
 
-## Basic requirements for a localization team
+ - [Initiate a new localization team](#initiate-a-new-localization-team)
+ - [Get involved in an existing localization team](#get-involved-in-an-existing-localization-team)
 
+Contributions to improving the following localization guides and localization policies are also welcome.
+
+---
+
+# Initiating a new localization team
+
+## Basic requirements to initiate a new localization team
+
+- The Glossary project does not have a localization team for your language yet
+  - check the language list from [localization progress](https://github.com/cncf/glossary/projects/2) and ask in the [glossary-localizations channel in Slack](https://cloud-native.slack.com/archives/C02N2RGFXDF). There may be a group of volunteers who haven't gotten started yet.
 - At least three volunteers
 - At least two approvers 
-  - Each term must be approved by two approvers
-  - Unanimous consent must be reached to be approved 
+  - two approvers must approve each term
+  - unanimous consent must be reached to be approved 
+  - at least one volunteer must have basic GitHub knowledge and be willing to help the others get up to speed (such as PR, review process, branch, ..)
 
 ## Instructions to initiate a new localization
 
-### Open an Issue to notify initiation of a new localization 
+### 1. Opening an issue to notify initiation of a new localization 
```

Comment:
We should also mention joining Slack. I think that may be the first step, what do you think?
****

# 1. Joining the Slack channel and saying hi

If you're interested in creating a new localization team, whether you have a team or are on your own, please join the #glossary-localizations channel on the CNCF slack and say hi. If you have at least three volunteers and are ready to start, request a localization channel for your language (e.g., #glossary-localization-korean). If you are still looking for more volunteers, let everyone in the channel know. There may be someone in there looking for a team as well.